app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  has_many :reservations
+  has_many :cars, through: :reservations
+
+  validates :username, presence: true
```

Comment:
Why don't add a minimal characters number? This could improve the user names. 🐴

Reply:
Fixed as advised. Thanks :relaxed: